hack/verify-code-patterns.sh

```diff
@@ -34,3 +35,11 @@ if [[ -n "$out" ]]; then
   echo >&2 "$out"
   exit 1
 fi
+
+# Do not use glog in test code
+out="$(grep --include '*_test.go' --exclude-dir 'vendor/' -EIrn 'github.com/golang/glog' || true)"
```

Comment:
I recommend making the pattern based on invocation `[kg]log\.` as we might move to klog at some point (in fact we probably should). Similarly, import paths are more likely to change than the usage (e.g. this could change to `golang.org/glog` someday).

Reply:
My thought was to avoid false positives as `glog` is rather short. But for now there are no collisions and we can think of a better way if a collision should occur. :+1: